### DISCUSSION TITLE
Snippets duplicates. How to config?

### DISCUSSION THREAD

**Discussion by milkpuff** [LazyVim/LazyVim#5142] [Q&A]
I enable lazyextra `coding.blink` and `coding.luasnip`:
![图片](https://github.com/user-attachments/assets/d88d16bd-b76a-47fe-a1c5-1823480368d8)
If disable `coding.luasnip`, snippets show once, but some snippet like `classi` raise error:
`/usr/share/nvim/runtime/lua/vim/lsp/_snippet_grammar.lua:177: snippet parsing failed`

**Comment by monoira** [comment]
https://github.com/LazyVim/LazyVim/issues/5210

I had same issue with repeating snippets so I opened issue with your discussion and it seems to be luasnip and blink compatibility issue.
replace luasnip with nvim-snippets. You could copy it from my dotfiles or wait for blink developers to fix it.
